core/src/date-input/date-input.tsx

```diff
@@ -34,6 +36,14 @@ interface Props {
 	 * Style of the text box. Choose one from `DateInput.styles`.
 	 */
 	style?: InputProps["style"];
+
+	disabled?: InputProps["disabled"];
+
+	maxDate?: Date;
+
+	minDate?: Date;
```

Comment:
This is good :+1: I was afraid that you would use the "modifier" type of react-day-picker, which is not good because it just doesn't make sense in "min - max" context